exporter/signalfxexporter/exporter.go

```diff
@@ -149,7 +153,6 @@ func (se *signalfxExporter) start(_ context.Context, host component.Host) (err e
 	}
 
 	dimClient := dimensions.NewDimensionClient(
-		context.Background(),
```

Comment:
Why do we need to remove context? Should we maybe propagate the existing context wrapped with `context.WithCancel(ctx)` and call the `se.cance()l` in the shutdown instead of the stop channel?

Reply:
makes sense, I can use that instead of a chan

Reply:
ok changes are much cleaner now, thanks!